src/components/Data/Collections/Create.vue

```diff
@@ -1,11 +1,86 @@
 <template>
   <div class="wrapper">
-    CollectionCreate
+    <headline>
+      Create a collection in <i>{{$route.params.index}}</i>
```

Comment:
Could we use props instead of accessing `$route`?
And thanks to vue-router-sync, in parent component, you could use the @stafyniaksacha getter that will provide you the current index in store (= index in url)